doc/SOUNDPACKS.md

```diff
@@ -22,23 +22,23 @@ Sound effects can be included with a format like this:
 
```

Comment:
I suggest running JSON examples through the [CDDA JSON formatter](https://dev.narc.ro/cataclysm/format.html). They may require putting some extra lines to make them a complete file, however.

Reply:
Done. Neat tool! It did produce one somewhat long line on [L72](https://github.com/CleverRaven/Cataclysm-DDA/pull/47523/files#diff-ae5ecf47213a1cc0a5651fe6f88f494a4f162e30e432c41139555a5fb31fbf77R72), but that's more of a nitpick.